account/client.go

```diff
@@ -1,6 +1,8 @@
 package account
 
 import (
+	"net/http"
```

Comment:
Just to possibly pre-empt some feedback: `net/http` is new in all client packages, but they were already depending on it transitively through the top-level `stripe` package, so I don't think including it in all of these will have any meaningful effect on final binary size.